extensions/amp-iframe/amp-iframe.md

```diff
@@ -26,6 +28,12 @@ limitations under the License.
 
 ## Usage
 
+[tip type="warning"]
+
+This component is work in progress.
+
+[/tip]
+
 Displays an AMP valid iframe. `amp-iframe` has several important differences from vanilla iframes that are
 designed to make it more secure and avoid AMP files that are dominated by a
 single iframe:
```

Comment:
Should the existing docs stay in this file? Should it be empty aside from the warning?

Reply:
Figured empty doc would look weird. That's why I added the warning.